### PR TITLE
Update couchbase tooling image

### LINKED ISSUE #1592: [BUG] Tooling images in Couchbase example BPs are outdated
**Describe the bug**

The tooling images we use in couchbase BPs are too old. We should update them to the latest released version so that the automation takes care of upgrading them automatically during the next release.
- https://github.com/kanisterio/kanister/blob/master/examples/couchbase/couchbase-blueprint.yaml#L22
- https://github.com/kanisterio/kanister/blob/master/examples/couchbase/couchbase-blueprint.yaml#L56
- https://github.com/kanisterio/kanister/blob/master/examples/couchbase/blueprint-v2/couchbase-blueprint.yaml#L61
- https://github.com/kanisterio/kanister/blob/master/examples/couchbase/blueprint-v2/couchbase-blueprint.yaml#L92

### PR DESCRIPTION
**Pull request by r4rajat**
## Change Overview

- Update couchbase tooling image in the couchbase blueprint.
- Update the pkg/app/couchbase.go for integration testing with respect to the latest couchbase release

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [ ] :hamster: Trivial/Minor
- [x] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [ ] :robot: Test

## Issues 

- fixes #1592 

## Test Plan

- Update the values of SHORT_APPS variable from build/integration-test.sh from ```SHORT_APPS="^PostgreSQL$|^MySQL$|Elasticsearch|^MongoDB$|Maria|^MSSQL$"``` to ```SHORT_APPS="^Couchbase$"```

- Install Minio
```$ bash build/minio.sh install_minio```

- Run following command
```$ bash build/integration-test.sh short```

Output:
```
Running integration tests:
~/go/src/github.com/kanisterio/kanister/pkg/testing ~/go/src/github.com/kanisterio/kanister
=== RUN   Test
{"File":"pkg/testing/integration_test.go","Function":"github.com/kanisterio/kanister/pkg/testing.(*IntegrationSuite).TestRun","Line":196,"app":"couchbase","level":"info","msg":"Running e2e integration test.","testName":"Couchbase.TestRun","time":"2022-08-12T13:51:11.422161807+05:30"}
{"File":"pkg/app/couchbase.go","Function":"github.com/kanisterio/kanister/pkg/app.(*CouchbaseDB).Install","Line":82,"app":"couchbase","level":"info","msg":"Installing couchbase operator and cluster helm chart.","time":"2022-08-12T13:51:11.450950415+05:30"}
.
.
.
{"ActionSet":"delete-test-actionset-mckwx-zzr4z","File":"pkg/controller/controller.go","Function":"github.com/kanisterio/kanister/pkg/controller.(*Controller).logAndSuccessEvent","Line":652,"level":"info","msg":"Updated ActionSet 'delete-test-actionset-mckwx-zzr4z' Status-\u003ecomplete","time":"2022-08-12T13:56:07.041087691+05:30"}
{"File":"pkg/app/couchbase.go","Function":"github.com/kanisterio/kanister/pkg/app.CouchbaseDB.Uninstall","Line":245,"app":"couchbase","level":"info","msg":"Uninstalling helm charts.","namespace":"couchbase-test","release":"couchbase-h8wqm","time":"2022-08-12T13:56:15.834441762+05:30"}
OK: 1 passed
--- PASS: Test (306.65s)
PASS

```

- [ ] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

File: pkg/app/couchbase.go [hunk 1]
```diff
@@ -39,6 +39,8 @@ const cbReadyTimeout = 5 * time.Minute
 // Regex to extract result from cb query response
 var countResp = regexp.MustCompile(`(?m){"\$1":([\d]+)},`)
 
+const numPings = 1
```
Comment (viveksinghggits) [inline pkg/app/couchbase.go:42]: this can be grouped with the const initialised in line 37.

**Review by viveksinghggits — APPROVED** [review]

File: pkg/app/couchbase.go (hunk 1 above)
Comment (r4rajat) [reply pkg/app/couchbase.go:45]: Done @viveksinghggits

File: pkg/app/couchbase.go [hunk 2]
```diff
@@ -129,7 +132,7 @@ func (cb *CouchbaseDB) Object() crv1alpha1.ObjectReference {
 	return crv1alpha1.ObjectReference{
 		APIVersion: "v2",
 		Group:      "couchbase.com",
-		Name:       fmt.Sprintf("%s-couchbase-cluster", cb.chart.Release),
+		Name:       fmt.Sprintf("%s", cb.chart.Release),
```
Comment (pavannd1) [inline pkg/app/couchbase.go:135]:
```suggestion
		Name:       cb.chart.Release,
```
This is already a string - will fix the lint failure

**Review by PrasadG193 — APPROVED** [review]

File: pkg/app/couchbase.go (hunk 2 above)
Comment (r4rajat) [reply pkg/app/couchbase.go]: Done @pavannd1

**Pull request merged**